ray-operator/apis/ray/v1alpha1/rayjob_types.go

```diff
@@ -0,0 +1,81 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+type JobStatus string
+
+const (
+	JobStatus_Pending   JobStatus = "PENDING"
```

Comment:
Let's remove `_` for consistency? `JobStatusPending` should be good

Reply:
+1